modules/facturacionAutomatica/schemas/factura.ts

```diff
@@ -0,0 +1,43 @@
+import * as mongoose from 'mongoose';
+
+export let schema = new mongoose.Schema({
```

Comment:
```suggestion
export let FacturaSchema = new mongoose.Schema({
```

¿Se usa este schema?

Reply:
No se usa